ui/Makefile

```diff
@@ -65,7 +67,9 @@ GOBINDATA_DEBUG_SOURCES := $(BOWER_COMPONENTS)/d3/d3.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.js \
                            $(BOWER_COMPONENTS)/moment/moment.js \
                            $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.js \
-                           $(GOBINDATA_FONTS)
+                           $(GOBINDATA_FONTS) \
+                           favicon.ico \
+                           apple-touch-icon.png
```

Comment:
favicon.ico is actually an ico file with 4 different sizes in it - 16x16, 32x32, 48x48 and, 256x256

The higher resolutions are for things like icons (eg how it appears in the windows taskbar) and 256x256 covers most of the edge cases, which is probably why it's included, but could probably be removed without affecting anything.

That said, favicons are cached very aggressively so it won't incur any cost after the first load and hedges our bets so we'll look good on any system where the icon appears.

I'm not a favicon expert, but this seems to be the idea behind the images from the blog.